com.unity.render-pipelines.universal/Shaders/Utils/StencilDeferred.shader

```diff
@@ -531,6 +550,7 @@ Shader "Hidden/Universal Render Pipeline/StencilDeferred"
             HLSLPROGRAM
 
             #pragma multi_compile _SSAO_ONLY
```

Comment:
Where does this keyword get enabled/disabled?

Reply:
The keyword never need to be enabled because it is the only option (there is no '_' option). It is equivalent to '#define _SSAO_ONLY', but the later does not work, I suppose because `_SSAO_ONLY` is tested in the `HLSLINCLUDE` block, which get added before the `#define _SSAO_ONLY` is actually parsed...